notebooks/_06-Architecture.md

```diff
@@ -21,33 +21,44 @@ jupyter:
 - **List needed intermediate products (e.g. pseudo-code, unit tests, etc.)**
 - **Try to be concise but comprehensive?**
 
-The project is split into two tasks:
-
-The first task will be based on simulation work and will go as follows:
-
-For a causal graph where all Xs are independent:
- - Ingest data
- - Manipulate data as needed, the output of this step will be a cleaned dataset
- - Specify a choice model based on the data at hand, the specification should follow required conventions by third party dependencies
- - The cleaned dataset will be used in a function that simulates choice data based on the specified model, resulting in a dataset with simulated choices
- - The dataset will be used to estimate a choice model and recover the necessary parameters. These parameters can be stored in a dictionary.
- - The previous two steps will be repeated for N times (say 100) and the parameters from each iteration will be stored in a dictionary entry.
- - A function that takes in the parameters of the estimated model and results in the percentage of time that the true parameter is recovered.
- - We will report parameter recovery percentages separately for each parameter, then build on that with aggregate percentaged for a subset of parameters 
-   then for all parameters.
+The project is split into three main tasks:
+
+1. The first task is focused on a selection-on-observables problem:
+
+For a causal graph with independent covariates:
+ 1. Ingest data
+ 2. Manipulate data as needed, the output of this step will be a cleaned dataset
+ 3. Specify a choice model based on the data at hand, the specification should follow required conventions by third party dependencies
```

Comment:
```suggestion
 3. Specify and estimate a choice model based on the data at hand, the specification should follow required conventions by third party dependencies
```

Without that change, steps 10 and 12 don't make sense (as we wouldn't have the "true" parameters that are estimated at step 3).

Reply:
Right!